backstop/tests/header.html

```diff
@@ -752,6 +752,362 @@ <h2>Slim</h2>
 
     <hr />
 
+    <h2>Split Menu Buttons</h2>
+    <section id="demo-split">
+
+      <header class="iui-page-header iui-slim">
+        <div class="iui-left">
+          <!-- Application Button -->
+          <div
+            class="iui-header-logo"
+            role="button"
+            aria-label="App name"
+            aria-expanded="false"
+            tabindex="0"
+          >
+            <!-- Use imodel-hollow.svg from @itwin/itwinui-icons package. -->
+            <svg
+              viewBox="0 0 16 16"
+              aria-hidden="true"
+              class="iui-header-button-icon"
+            >
+              <path
+                d="M8,1.0002a1.00413,1.00413,0,0,1,.51376.13829L13.52436,4.119A.95115.95115,0,0,1,14,4.93885v6.0975a1.00386,1.00386,0,0,1-.48626.85077l-5,2.9744a1.02348,1.02348,0,0,1-1.02748,0l-4.98453-2.965A1.03179,1.03179,0,0,1,2,11.03635V4.96365a1.004,1.004,0,0,1,.48624-.85086l5-2.97431A1.00412,1.00412,0,0,1,8,1.0002m0-1A2.00636,2.00636,0,0,0,6.975.27905l-5,2.9743A1.9939,1.9939,0,0,0,1,4.96365v6.0727a2.025,2.025,0,0,0,.975,1.7102l5,2.9744a2.02328,2.02328,0,0,0,2.05,0l5-2.9744A1.99376,1.99376,0,0,0,15,11.03635V4.93885a1.95338,1.95338,0,0,0-.975-1.6855l-5-2.9743A2.00636,2.00636,0,0,0,8,.0002ZM8,3.25005a1.0625,1.0625,0,1,1-1.0625,1.0625h0A1.0625,1.0625,0,0,1,8,3.25005Zm3,9.1894a.25006.25006,0,0,1-.1221.2148l-2.725,1.6229a.24974.24974,0,0,1-.255.0004L5.1229,12.639A.24992.24992,0,0,1,5,12.42365v-1.4408a.24991.24991,0,0,1,.3778-.2148L7,11.66667V8.23635l-.8778-.5593A.25.25,0,0,1,6,7.46215V6.01635a.25.25,0,0,1,.3783-.2146l2.5,1.3831A.24988.24988,0,0,1,9,7.39935v4.26732l1.6222-.89862a.25.25,0,0,1,.3778.2148Z"
+              />
+            </svg>
+            <span class="iui-label">App name</span>
+          </div>
+
+          <div class="iui-divider"></div>
+
+          <!-- Project Button -->
+          <nav aria-label="breadcrumbs">
+
+            <span class='iui-button iui-borderless iui-header-button iui-split-menu-header-button'>
+              <button
+                class="iui-button iui-borderless iui-header-button iui-large"
+                type="button"
+                aria-label="Project picker"
+                aria-expanded="false"
+                id="test-button-5"
+              >
+                <div class="iui-icon iui-header-button-icon demo-picture"></div>
+                <span class="iui-label">
+                  <div>Project Alpha</div>
+                  <div class="iui-description">0x0123456789</div>
+                </span>
+                <!-- Use caret-down-small.svg from @itwin/itwinui-icons package. -->
+
+                <span
+                  id='test-button-7'
+                  class="iui-button iui-borderless iui-header-button iui-large iui-split-menu"
+                >
```

Comment:
Can you add `tabindex="0"` to the split menu buttons so that we can test for `:focus`?
![Screen Shot 2021-07-28 at 5 16 23 PM](https://user-images.githubusercontent.com/849817/127397127-17d9b38c-5671-4c0c-b4ab-5ed4bd267828.png)

Reply:
Added tabIndex

Reply:
tabindex wont be necessary if we use button instead of span

Reply:
[Nesting a button within a button doesn't seem valid.](https://stackoverflow.com/a/39386695)

Reply:
What if we do a button within an anchor link?  Clicking on the anchor link takes you to a page about the project / model, clicking on the nested button opens the dropdown menu.

Reply:
I think this comment can be resolved as we are using button